CMakeLists.txt

```diff
@@ -2,6 +2,10 @@ cmake_minimum_required(VERSION 3.2.0)
 
 project(Poco)
 
+if(WINCE AND MINGW)
+    message(FATAL_ERROR "MinGW does not support WinCE")
```

Comment:
I think I should delete this. CMake will cope with this.

Reply:
I think also we should remove this.... mingw will not work at all on wince ...